build.fsx

```diff
@@ -107,6 +107,15 @@ Target "BuildApp" (fun _ ->
         |> DoNothing
 )
 
+Target "BuildDotNetCore" (fun _ ->
+    Fake.DotNetCli.Restore (fun p ->
+      {p with
+          WorkingDir = "./Octokit.Next" })
+
+    !! "./Octokit.Next/project.json"
+    |> Fake.DotNetCli.Build (fun p -> p)
```

Comment:
maybe there's an easier way than `(fun p -> p)` to return the input parameter as is